doc/Doxyfile.extra.in

```diff
@@ -8,3 +8,5 @@ ALIASES   += Link{1}="\ref \1"
 ALIASES   += Link{2}="\ref \1 \"\2\""
 ALIASES   += LHPP{2}="\Link{hpp::\1::\2,\2}"
 ALIASES   += LPinocchio{1}="\LHPP{pinocchio,\1}"
+
+EXTRA_PACKAGES = {amsmath}
```

Comment:
```suggestion
```

The cmake module add this and another package.

Reply:
Done in 85b4f26.